Offices/Offices.Tests/DoctorServiceTests.cs

```diff
@@ -0,0 +1,314 @@
+using AutoFixture;
+using AutoMapper;
+using MongoDB.Driver.Core.Misc;
+using Moq;
+using Offices.Application.MappingProfiles;
+using Offices.Application.Models;
+using Offices.Application.Services;
+using Offices.Data.Entities;
+using Offices.Data.Repositories.Abstractions;
+
+namespace Offices.Tests
+{
+    public class DoctorServiceTests
+    {
+        private readonly IMapper _mapper;
+
+        public DoctorServiceTests()
+        {
+            var configuration = new MapperConfiguration(cfg =>
```

Comment:
Creating profile for unit tests can be considered as an option, but let's leave unit tests fully abstract and mock mapper as well. We'll create mapper config for integration tests

Reply:
Fixed